src/PlatformInterface.php

```diff
@@ -9,6 +9,12 @@
 
 interface PlatformInterface extends CommandQuestionInterface {
 
+  const PLATFORM_TYPE_ID = 'platform.type';
+
+  const PLATFORM_NAME = 'platform.name';
```

Comment:
Cannot inherit previously-inherited or override constants, implementing classes will fail, they need to be renamed.

Reply:
Done